src/components/Capture/Face.js

```diff
@@ -77,29 +77,29 @@ class Face extends Component {
       ...props,
     }
 
-    return useWebcam && hasCamera ?
-      requestedVariant === 'video' ?
-        <Video
-          {...cameraProps}
-          onVideoCapture={ this.handleVideoCapture }
-        /> :
-        <Selfie
-          {...cameraProps}
-          onCapture={ this.handleCapture }
-          useMultipleSelfieCapture={ useMultipleSelfieCapture }
-          snapshotInterval={ snapshotInterval }
-        />
-      :
-      props.uploadFallback ?
+    return (
+      useWebcam && hasCamera ?
+        requestedVariant === 'video' ?
+          <Video
+            {...cameraProps}
+            onVideoCapture={ this.handleVideoCapture }
+          /> :
+          <Selfie
+            {...cameraProps}
+            onCapture={ this.handleCapture }
+            useMultipleSelfieCapture={ useMultipleSelfieCapture }
+            snapshotInterval={ snapshotInterval }
+          /> :
+      this.props.uploadFallback ?
         <Uploader
           {...props}
           onUpload={ this.handleUpload }
           title={ translate('capture.face.upload_title') || title }
           instructions={ translate('capture.face.instructions') }
-          />
-      :
-        <GenericError error={{name: 'GENERIC_CLIENT_ERROR'}}/>
+          /> :
+          (!isDesktop || this.props.mobileFlow) && <GenericError error={{name: 'INTERRUPTED_FLOW_ERROR'}} />
```

Comment:
So desktop users just get... an empty screen?

Reply:
Desktop users are redirected to x-device if camera cannot be used

Reply:
Ah, yes

In that case - why do these checks at all? Why not just omit the `(!isDesktop || this.props.mobileFlow)` bit and always render the error at this bit?

Maybe with a comment about the x-device flow for desktop

Reply:
Just me over-engineering things!